storageservice/modules/files/file_client.bal

```diff
@@ -240,6 +240,52 @@ public isolated client class FileClient {
             azureDirectoryPath);
     }
 
+    # Gets File Metadata.
+    #
+    # + fileShareName - Name of the FileShare  
+    # + fileName - Name of the File  
+    # + azureDirectoryPath - Path of the azure directory
+    # + uriParameters - Optional URI Parameter Description
+    # + return - If successful, File Metadata. Else an Error
+    @display {label: "Get File Metadata"}
+    remote isolated function getFileMetadata(@display {label: "File Share Name"} string fileShareName,
+                                             @display {label: "File Name"} string fileName, 
+                                             @display {label: "Azure Directory Path"}string? azureDirectoryPath = (),
+                                             @display {label: "Optional Parameters"} OptionalURIParametersFileMetaData
+                                             uriParameters = {}) returns @display {label: "File metadata"} 
```

Comment:
`OptionalURIParametersFileMetaData ` does this get properly rendered in Choreo lowcode?

Reply:
Yes, this is similar to the other function's option URL parameter inputs records

Reply:
![image](https://user-images.githubusercontent.com/25779551/157442743-83a3541a-75a2-484d-862b-94fc2cf96938.png)